pkg/controller/bootstrap/bootstrap.go

```diff
@@ -79,7 +83,10 @@ func (b *Bootstrap) Run(destDir string) error {
 		}
 
 		for idx, m := range manifests {
-			obji, err := runtime.Decode(scheme.Codecs.UniversalDecoder(v1.SchemeGroupVersion), m.Raw)
+			obji, err := runtime.Decode(scheme.Codecs.UniversalDecoder(v1.GroupVersion), m.Raw)
```

Comment:
This is how installer uses a decoder for multiple schemes https://github.com/openshift/installer/blob/6e2977c740853e842247b55c7cd08d9b350f3e93/pkg/asset/machines/master.go#L287-L301

I think we should do the same rather than handling `runtime.IsNotRegisteredError(err)`